api-specificatie/openapi.yaml

```diff
@@ -769,6 +702,19 @@ components:
       schema:
         type: string
 
+    locatie-query:
+      description: Coordinaten van een locatie die als query-parmeter gebruikt wordt om een object te zoeken. Let op, explode is false dus het formaat is ?locatie=1.2345,5.4321
```

Comment:
waarom heb je niet ook een example opgenomen? daarvoor hebben we keyword example, dus dat hoeft niet in de description.

Reply:
Dat kan ik doen, maar ik weet niet of dat het goede beeld geeft. locatie is namelijk een array, maar ik kan maar 1 example waarde opgeven. Die wordt in de swagger-view niet getoond (heeft denk ik te maken met dat het een array is). 

Als daar een voorbeeld uit te genereren is dan zou dat ?locatie=1.2345  worden terwijl het essentieel is dat er 2 coordinatem meegegeven worden. Dat kan ik in de description wel zo opnemen.
@MelvLee hoe kijk jij daar tegenaan ?

Reply:
je kan op drie plekken een example meegeven, op het niveau van de operatie, op het niveau van de parameter en op het niveau van het array-item. Wat je beschrijft is een example op niveau van het array-item. Ik zou dit doen op het niveau van de parameter (zoals we dat ook gedaan hebben voor burgerservicenummer in BRP:
    locatie-query:
      description: Coordinaten van een locatie die als query-parmeter gebruikt wordt om een object te zoeken. Let op, explode is false dus het formaat is ?locatie=1.2345,5.4321
      name: locatie
      in: query
      required: false
      explode: false
      example: '194602.722,464154.308'
      schema:
        type: array
        minItems: 2
        items:
          type: number

Klopt dat in Swagger het voorbeeld niet wordt getoond, maar dat neemt niet weg dat een example altijd goed is. Maar je hebt gelijk dat dit een reden is het voorbeeld ook in de description op te nemen.

De waarden in het voorbeeld die je geeft kunnen trouwens ook verwarrend zijn, die liggen ergens in Frankrijk. Bijvoorbeeld  '194602.722,464154.308' ligt wel in Nederland.

Reply:
Je kan een example voor een array als volgt definieren

![image](https://user-images.githubusercontent.com/18151729/80798096-cf2d3d00-8ba3-11ea-808b-b45fdfc84f4a.png)

Dan ziet het in de swagger-view er als volgt uit:

![image](https://user-images.githubusercontent.com/18151729/80798167-ff74db80-8ba3-11ea-89da-d0a3ed2350a6.png)

Ik weet alleen niet of het dan duidelijk genoeg is. Voor mij nog niet (misschien moet ik er nog aan wennen)

Reply:
de manier van opnemen zoals Melvin toont (als array) is werkt in ieder geval in Swagger wel het beste. Swagger kan dan met Try out laten zien hoe dit er in curl uitziet. Aangevuld met een voorbeeld in de toelichting moet dit toch duidelijk zijn?

Reply:
Lijkt mij ook Ik heb Melvins methodiek en de waarden die Frank heeft voorgesteld overgenomen.